### PR TITLE
Fix/contro width of tasks page and task

### PR DESCRIPTION
**Pull request by hilabar1**

### CODE REVIEW

**Comment by hilabar1** [comment]
this is a fix for:
https://trello.com/c/lsrvfCEI/142-create-a-long-tasks-breaks-the-view-for-all-users

**Comment by rootux** [comment]
@hilabar1  That doesn't work for me

It still overlap outside of the card itself
Screen Shot 2019-11-15 at 13 39 38

Open this project and scroll to the end of it to see the above
http://localhost:3000/testyoni/task/1?complete=false

**Comment by rootux** [comment]
@hilabar1 
Ok. I know why it looked like it was fixed.
That would work for a long sentances seperated by words with space. But not for a long no spaces words for any of those fields (For example - "Testing123" vs. "Testing 123"

**Comment by hilabar1** [comment]
> @hilabar1
> Ok. I know why it looked like it was fixed.
> That would work for a long sentances seperated by words with space. But not for a long no spaces words for any of those fields (For example - "Testing123" vs. "Testing 123"

I actually tested this scenario, but anyway, once i merged with master again it was broken.
Anyway, I fixed everything, also tested on safari. looks ok.
Please let me know what you think :)

**Comment by rootux** [comment]
Looks good.
It doesn't break the text now. But the view still gets broken - 
Open this - 
http://localhost:3000/testyoni/task/1

Scroll all the way down - View gets broken

(Only this time since overflow is hidden we can't scroll to it - only see all if we zoom out to a small screen size :) )

File: src/views/molecules/taskViewMini/taskViewMini.scss [hunk 1]
```diff
@@ -28,6 +28,10 @@
     .label-wrapper {
       background-color: white !important;
       color: $primary-color !important;;
+      white-space: nowrap;
```
Comment (rootux) [inline src/views/molecules/taskViewMini/taskViewMini.scss]: This is set under the &.active - which is only for the active task.

**Pull request merged**